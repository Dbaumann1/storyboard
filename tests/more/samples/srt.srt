1
00:00:01,000 --> 00:00:02,000
SubRip is the way to go
