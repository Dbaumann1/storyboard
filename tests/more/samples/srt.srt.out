Filename:               srt.srt
File size:              56 (56B)
SHA-1 digest:           36bdfdadefbc73384ef20911aadef3b843c12330
Container format:       SRT
Duration:               Not available
Streams:
    #0: Subtitle, SubRip

